G4EMPH/BeamGen_module.cc

```diff
@@ -275,15 +275,16 @@ namespace emph {
     // now get beam particle position
     TLorentzVector pos;
     
-    pos[2] = fZstart;
+    pos[2] = fZstart; // units are mm for this
+
     if (fXYHist) { // get random position from histogram
       fXYHist->GetRandom2(pos[0],pos[1]);
     }
     else { // get random position from flat or Gaussian distribution
       if (fXYDistSource == "FlatXY" || fXYDistSource == "flatXY" ||
```

Comment:
make fhicl-configurable?

Reply:
"Zstart" is a fhicl-config.  Is that what you mean, or do you mean the units?

Reply:
ah ignore this one, i mis-read things.